sklearn/linear_model/_least_angle.py

```diff
@@ -963,6 +970,13 @@ def fit(self, X, y, Xy=None):
         else:
             max_iter = self.max_iter
 
+        if self.jitter:
+            noise = np.random.RandomState(0).uniform(high=self.jitter,
+                                                     size=len(y))
```

Comment:
any use of random number generator should use a RandomState and the random_state should be a parameter of the estimator.